### DISCUSSION TITLE
A discussion got closed without explanation, most likely mistakenly

### DISCUSSION THREAD

**Discussion by marcindulak** [community/community#123987] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The discussion https://github.com/orgs/community/discussions/2971, started in 2020, has been recently marked as answered, and closed by @LiteBrite82.

In the opinion of the discussion participants it happened mistakenly, as this is the discussion which acquired most upvotes on the given topic. Despite requests from participants, there is no reponse or action from @LiteBrite82 yet. Please reopen the discussion, so it may get onto the feature roadmap.

**Comment by LiteBrite82** [comment]
Hi @marcindulak 

I closed it in order to consolidate and refresh the topic; old posts can be overlooked for feature requests. I want to make sure the dev team knows that there is still interest in the topic.

**Reply by marcindulak** [reply]
As explained in https://github.com/orgs/community/discussions/2971#discussioncomment-9485711, the newer, duplicate discussion https://github.com/orgs/community/discussions/3211, from January 22 2021, is only about a month newer than https://github.com/orgs/community/discussions/2971, from December 12 2020, which got closed. I would say the (concerning) approach of focusing on newer issues does not apply here.
